src/grammar/cue.bnf

```diff
@@ -219,6 +221,13 @@ private mul_op     ::= MUL_OP
 // "+" | "-" | "!" | "*" | rel_op
 private unary_op   ::= ADD_OP | EXCL | "*" | rel_op
 
+UnificationBinaryExpr ::= Expression OP_UNIFICATION Expression  {
```

Comment:
If you'd like to keep this, please make it `extend=BinaryExpr`. If you also move the `methods=` part to `BinaryExpr`, then the other code referencing left and right will work without any other changes.